maxim/logger/components/utils.py

```
@@ -0,0 +1,81 @@
+                    # Check if its base64 encoded data uri
+                    if url.startswith("data:image"):
+                        # Extract base64 data from data URI
+                        match = re.match(
+                            r"data:image/(?P<ext>\w+);base64,(?P<data>.+)", url
+                        )
+                        if match:
+                            ext = match.group("ext")
+                            data = match.group("data")
+                            try:
+                                file_data = base64.b64decode(data)
+                            except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)